env_file_app_template

```diff
@@ -45,6 +45,7 @@ PULSEDIVE_API_KEY=
 INTELX_API_KEY=
 WHOISXMLAPI_KEY=
 UNPAC_ME_API_KEY=
+URLSCAN_KEY=1a891838-6834-498e-a7cb-729e920c4f9c
```

Comment:
You should never push your personal API key!! 
Also, the variable name is wrong. It should be `URLSCAN_API_KEY` as declared by you in `urlscan.py` and `analyzer_config.json`

The line should just be: `URLSCAN_API_KEY=`